include/odp/api/spec/packet_flags.h

```diff
@@ -23,87 +23,114 @@ extern "C" {
 #include <odp/api/packet.h>
 
 /** @addtogroup odp_packet
- *  Boolean operations on a packet.
+ *  Operations on packet metadata flags.
  *  @{
  */
 
 /**
- * Check for packet errors
+ * Check for all errors in packet
  *
- * Checks all error flags at once.
+ * Check if packet parsing have found any errors in the packet. The level of
+ * error checking depends on parse configuration (e.g. included layers and
+ * checksums). Protocol layer functions (e.g. odp_packet_has_l3()) indicate
+ * which layers have been checked, and layer error functions
+ * (e.g. odp_packet_has_l3_error()) which layers have errors.
  *
- * @param pkt Packet handle
- * @retval non-zero packet has errors
- * @retval 0 packet has no errors
+ * @param pkt          Packet handle
+ *
+ * @retval non-zero    Packet has errors
+ * @retval 0           No errors were found
  */
 int odp_packet_has_error(odp_packet_t pkt);
 
 /**
- * Check for packet L2 errors
+ * Check for errors in layer 2
  *
- * check for all L2 errors
+ * Check if packet parsing have found any errors in layer 2 of the packet.
+ * Make sure that the layer is included into parse configuration.
```

Comment:
By context  this seems to be an input requirement usage note, so a clearer statement might be "If layer 2 is included in the parse configuration, check if packet parsing has found any errors in layer 2 of the packet".

Reply:
OK. Changed to: "When layer 2 is included in the parse configuration, check if any errors were found in layer 2 of the packet."